src/NUnitFramework/framework/Attributes/MaxTimeAttribute.cs

```diff
@@ -47,7 +47,8 @@ public MaxTimeAttribute( int milliseconds )
 
         #region ICommandWrapper Members
 
-        TestCommand ICommandWrapper.Wrap(TestCommand command)
+        /// <inheritdoc />
```

Comment:
Inheritdoc does nothing without a custom XML post-processing step which we currently don't have.

Reply:
yeah, in the old days of Sandcastle, that was the way to handle this, as I don't like the idea to duplicate all interfaces comments as keeping them in sync after is laborious, but if it is the way it is done in NUnit, I will change it.

Reply:
@xoofx It's not a matter of a decision NUnit has made but simply that it doesn't work for Intellisense. It's academic, however, unless we make the change.